src/libs/NetworkConnection.ts

```diff
@@ -72,42 +73,51 @@ Onyx.connect({
  * internet connectivity or not. This is more reliable than the Pusher
  * `disconnected` event which takes about 10-15 seconds to emit.
  */
-function subscribeToNetInfo(): void {
-    // Note: We are disabling the configuration for NetInfo when using the local web API since requests can get stuck in a 'Pending' state and are not reliable indicators for "offline".
+function subscribeToNetInfo() {
+    let backendReachabilityCheckInterval: NodeJS.Timeout;
```

Comment:
Would you have something against making this `const` + ternary?

Reply:
`const backendReachabilityCheckInterval = CONFIG.IS_USING_LOCAL_WEB ? setInterval(...) : undefined;`

Reply:
But, once we do this, we can ask ourselves if extracting `setInterval(...)` to something like `subscribeToBackendReachability` (a new file-local function) isn't more readable.

Reply:
It's a step in the right direction, in my opinion. If a variable is declared `let`, when reading the code, you have to be open to the fact that it's mutated in any other place of the scope. So it's more readable on a higher level of abstraction.

If you are unsatisfied with how the code lays out visually after this change, you can extract the inner block to a separate function, as I mentioned before. You can extract `setInterval(...)` expression to something like subscribeToBackendReachability (a new file-local function).

We'll end up with a short and enclosed ternary expression.

Reply:
Also, extracting parts of the code is good for other reasons, so I think it would be great if we did both things (switching to `const` for `backendReachabilityCheckInterval` and moved the code block to a separate function).